codec-compression/src/test/java/io/netty/handler/codec/compression/ZstdEncoderTest.java

```diff
@@ -75,6 +88,22 @@ public void testCompressionOfSmallBatchedFlow(final ByteBuf data) throws Excepti
         testCompressionOfBatchedFlow(data);
     }
 
+    @ParameterizedTest
```

Comment:
Just use `@Test` as you only will execute this with one input anyway

Reply:
Done